presto-docs/src/main/sphinx/admin/dynamic-filtering.rst

```diff
@@ -211,3 +211,7 @@ Limitations
 * Dynamic filtering is currently implemented only for :doc:`/connector/hive` and :doc:`/connector/memory` connectors.
 * Push down of dynamic filters into local table scan on worker nodes is limited to broadcast joins.
 * Min-max dynamic filter collection is not supported for DOUBLE, REAL and unorderable data types.
+* Dynamic filtering is supported in limited scenarios when join keys are of different types.
```

Comment:
Maybe rephase `Dynamic filtering is supported when there is join key cast from build key type to probe key type. Dynamic filtering is also supported in limited scenarios when there is implicit cast from probe key type to build key type. For example, dynamic ...`

Reply:
updated